Resources/Prototypes/_Gabystation/Loadouts/loadout_groups.yml

```
@@ -29,3 +30,12 @@
   loadouts:
   - GoldenShaker
   - DrinkShaker
+
+- type: loadoutGroup
+  id: TrinketsChef
+  name: loadout-group-trinkets-job
+  minLimit: 0
+  maxLimit: 2
```

```suggestion
  maxLimit: 1
```
Acho mais legal eles poderem escolher entre um ou outro